packages/graphiql-react/src/editor/style/codemirror.css

```diff
@@ -34,6 +34,9 @@
  */
 
 .cm-s-graphiql {
+  /* Default to punctuation */
+  color: var(--color-neutral-60);
```

Comment:
how does this work when other codemirror themes are introduced? Would it be easy for them to override this using css variables to suit a different theme if they didn't want to impact the rest of the styling? 

for example, a light graphiql theme with a dark codemirror theme, or vice versa?